node/node.go

```diff
@@ -393,6 +394,42 @@ func (n *Node) GetBalance(address string) (*big.Int, error) {
 	return account.GetBalance(), nil
 }
 
+// GetValueForKey will return the value for a key from a given account
+func (n *Node) GetValueForKey(address string, key string) (string, error) {
+	keyBytes, err := hex.DecodeString(key)
+	if err != nil {
+		return "", fmt.Errorf("invalid key: %w", err)
+	}
+
+	if check.IfNil(n.addressPubkeyConverter) || check.IfNil(n.accounts) {
```

Comment:
maybe you can make a method that return the account ... because GetValueForKey do more than one think ... gets the account and do a lot of checks and after that get value for key ....

Reply:
I would leave it like this